libs/sdk/e2e-schematics/src/generators/configure-storybook/index.spec.ts

```diff
@@ -40,6 +40,7 @@ describe('configure-storybook', () => {
     await createTestApplication(tree, {
       name: `test-app`,
       e2eTestRunner: true,
+      unitTestRunner: true,
```

Comment:
Changes:
* add `prebuilt` e2e configurations
* reduce the complexity by breaking up into multiple functions and removing unnecessary conditionals